content/en/docs/Getting started/_index.md

```diff
@@ -13,82 +13,132 @@ description: >
 
 ### Install Hugo 
 
-You need a [recent version](https://github.com/gohugoio/hugo/releases) of Hugo to do local builds and previews of sites (like this one) that use Docsy. If you install from the release page, make sure to get the `extended` Hugo version, which supports SCSS: you may need to scroll down the list of releases. Hugo can be installed via Brew if you're running MacOs. If you're a Linux user, do not use `sudo apt-get install hugo`, as it currently doesn't get you the `extended` version.
+You need a [recent version](https://github.com/gohugoio/hugo/releases) (we recommend version 0.54.0 or later) of [Hugo](https://gohugo.io/) to do local builds and previews of sites (like this one) that use Docsy. If you install from the release page, make sure to get the `extended` Hugo version, which supports [SCSS](https://sass-lang.com/documentation/file.SCSS_FOR_SASS_USERS.html); you may need to scroll down the list of releases to see it. 
```

Comment:
It actually works fine with 0.53 extended - how about:

You need a [recent "extended" version]() (we recommend version 0.53 or later)...

Reply:
done.